modules/fx/arrow-fx/src/main/kotlin/arrow/fx/extensions/schedule.kt

```diff
@@ -43,45 +43,39 @@ interface ScheduleApplicative<F, Input> : Applicative<SchedulePartialOf<F, Input
   fun MF(): Monad<F>
 
   override fun <A> just(a: A): Kind<SchedulePartialOf<F, Input>, A> =
-    Schedule.forever<F, A>(MF()).const(a) as Schedule<F, Input, A>
+    Schedule.once<F, A>(MF()).const(a) as Schedule<F, Input, A>
 
   override fun <A, B> Kind<SchedulePartialOf<F, Input>, A>.map(f: (A) -> B): Kind<SchedulePartialOf<F, Input>, B> =
     fix().map(f)
 }
 
 @extension
 interface ScheduleSemigroup<F, Input, Output> : Semigroup<Schedule<F, Input, Output>> {
-  fun SA(): Semigroup<Output>
+  fun OI(): Semigroup<Output>
 
   override fun Schedule<F, Input, Output>.combine(b: Schedule<F, Input, Output>): Schedule<F, Input, Output> =
-    and(b).map { (a, b) -> SA().run { a + b } }
+    and(b).map { (a, b) -> OI().run { a + b } }
 }
 
 @extension
 interface ScheduleMonoid<F, Input, Output> : Monoid<Schedule<F, Input, Output>>, ScheduleSemigroup<F, Input, Output> {
-  override fun SA(): Semigroup<Output> = MA()
-  fun MA(): Monoid<Output>
+  override fun OI(): Monoid<Output>
   fun MF(): Monad<F>
 
   override fun empty(): Schedule<F, Input, Output> =
-    Schedule.forever<F, Input>(MF()).const(MA().empty())
+    Schedule.once<F, Input>(MF()).const(OI().empty())
```

Comment:
Same here as with the applicative one. This cannot be lawful as `once` is not an identity element. If our tests cannot uncover this, something is wrong. Likely with Schedules eqK or genK instances.